libraries/DataFlash/DataFlash_Backend.h

```diff
@@ -154,9 +154,6 @@ class DataFlash_Backend
     DFMessageWriter_DFLogStart *_startup_messagewriter;
     bool _writing_startup_messages;
 
-    uint32_t _internal_errors;
-    uint32_t _dropped;
```

Comment:
_internal_errors is changing now to uint8_t.  I'd prefer to keep the `_` prefix since it's protected and the original type.

Reply:
I think there's been general agreement to lose the `_`'s, thus my choice of nuking this one.

As for the type - something serious has gone wrong if that ever moves value - so think of it as a boolean on steroids.

Reply:
Not that I know of. And it's there on our coding style page. It's good to have consistency even if it's not the coding style we love.